projects/plugins/jetpack/extensions/plugins/seo/index.js

```diff
@@ -99,7 +99,7 @@ const Seo = () => {
 	const hasRequiredPlanForEnhancer = ! getRequiredPlan( 'ai-seo-enhancer' );
 
 	const jetpackSeoPanelProps = {
-		title: __( 'SEO', 'jetpack' ),
+		title: __( 'Optimize SEO', 'jetpack' ),
```

Comment:
I am not sure we are supposed to update these panels.

Reply:
Thanks, I checked with the design and it's ok.